config/jobs/kubernetes/enhancements/enhancements-presubmit.yaml

```diff
@@ -1,11 +1,12 @@
 presubmits:
   kubernetes/enhancements:
-  - name: pull-enhancements-verify
-    always_run: false  # TODO(Katharine,nikhita): Turn this back on when the job works.
-    decorate: true
-    spec:
-      containers:
-      - image: golang:1.11
+    - name: pull-enhancements-verify
+      always_run: false # TODO(Katharine,nikhita): Turn this back on when the job works.
```

Comment:
I intend to flip this after manually triggering this on a (dummy) PR on the repo and verifying this works.